Manage data - New Search Filter - Screenshots updated (#4246)
SumoLogic/sumologic-documentation: docs/manage/partitions/edit-data-forwarding-destinations-partition.md

You can specify data forwarding settings for a partition so that the messages that were routed to an index can be forwarded to an existing or new Amazon S3 destination.

1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu select **Manage Data > Logs > Partitions**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Logs** select **Partitions**. You can also click the **Go To...** menu at the top of the screen and select **Partitions**. Kanso--> 

> 1. To refine the table results, utilize the **Add a filter** section located above the table. *AND* logic will be applied if you are filtering between different sections and *OR* logic will be applied if you are filtering between the same section. Click the Partition you want to update.

Author: ```suggestion
1. To refine the table results, use the **Add a filter** section located above the table. *AND* logic is applied when filtering between different sections, while *OR* logic is applied when filtering within the same section. Click the Partition you want to update.
```
